package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "10.4.3",
     "@testing-library/user-event": "12.0.11",
     "@types/codemirror": "0.0.96",
+    "@types/deep-equal": "^1.0.1",
```

Comment:
Do we want to pin those new dependency directly?

Reply:
done

Reply:
there are quite a few more unpinned?

Reply:
fixed